src/main/java/frc/robot/Robot.java

```diff
@@ -91,34 +100,21 @@ public void autonomousInit() {
     }
   }
 
-  /**
-   * This function is called periodically during autonomous.
-   */
   @Override
   public void autonomousPeriodic() {
     CommandScheduler.getInstance().run();
   }
 
   @Override
   public void teleopInit() {
-    this.teleopCommand = new TeleopCommand(controls, drive);
-    if (teleopCommand != null) {
-      teleopCommand.schedule();
-    }
```

Comment:
This comment: https://github.com/PearceRobotics/infinite-recharge-2020/pull/54/files#r379175534 applies here as well.